schema/traces/README.md

```diff
@@ -0,0 +1,104 @@
+* Parent's Span identifier.
+* Links to causally-related Spans (via the SpanContext of those related Spans).
+* SpanContext information required to reference a Span. 
+
+### SpanContext
+Represents all the information that identifies Span in the Trace and is propagated to child Spans and across process boundaries.
+A **SpanContext** contains the tracing identifiers and the options that are propagated from parent to child Spans.
+
+* TraceId  - It is worldwide unique with practically sufficient probability by being made as 16 randomly generated bytes - used to group all spans for a specific trace together across all processes. 
+* SpanId  -  It is the identifier for a span, globally unique with practically sufficient probability by being made as 8 randomly generated bytes. When passed to a child Span this identifier becomes the parent span id for the child Span.
+* Tracestate - carries tracing-system specific context in a list of key value pairs. Tracestate allows different vendors propagate additional information and inter-operate with their legacy Id formats. For more details see this.
+
+Additional fields can be supported via the Attributes key/value store see [traces](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md)
+
+### Structure
+The default fields that are supported by the traces are
+- **TraceId**  : It is worldwide unique with practically sufficient probability by being made as 16 randomly generated bytes - used to group all spans for a specific trace together across all processes.
+- **SpanId**   : It is the identifier for a span, globally unique with practically sufficient probability by being made as 8 randomly generated bytes. When passed to a child Span this identifier becomes the parent span id for the child Span.
+- **ParentId** : It is the identifier for a span's parent span.
+- **TraceState** : carries tracing-system specific context in a list of key value pairs. Tracestate allows different vendors propagate additional information and inter-operate with their legacy Id formats. For more details see this.
+
+- **Name**   :  String representing the span's name   
+- **Kind** 
+  - SpanKind.CLIENT	
+  - SpanKind.SERVER	
+  - SpanKind.CONSUMER
+  - SpanKind.PRODUCER
+  - SpanKind.INTERNAL
+
+- **StartTime** : Start time of the event
+- **EndTime**   : End time of the event
+- **Attributes**
+  - An Attribute is a key-value pair, which has the following structure  [Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/b00980832b4b823155001df56dbf9203d4e53f98/specification/common/README.md#attribute)
+
+- **DroppedAttributesCount** : Integer counting the dropped attributes
+- **Events**     : A set of the next tuples (timestamp, name, Attributes)
+- **DroppedEventsCount** : Integer counting the dropped events
+- **Links**  : links to causally-related Spans
+- **DroppedLinksCount** : Integer counting the dropped links
+- **Status** - 
```

Comment:
I don't know if you need to include it here, but maybe separate out status.code and status.message?

Reply:
will add the needed description...